debezium-connector-binlog/src/test/resources/ddl/innodb_mvcc.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TABLE test_table (
+					id INT AUTO_INCREMENT PRIMARY KEY,
+					name VARCHAR(255),
+					email VARCHAR(255),
+					last_updated TIMESTAMP);
+
+INSERT INTO test_table (name,email,last_updated) VALUES ('John Doe1','johndoe@gmail.com',NOW());
```

Comment:
IT nerds start counting with 0, no? ;))
btw, what are you doing with my email address here?

(don't take this comment serious)